src/components/NcPopover/NcPopover.vue

```diff
@@ -126,10 +126,12 @@ export default {
 		/**
 		 * Set element to return focus to after focus trap deactivation
 		 *
-		 * @type {import('focus-trap').Options['setReturnFocus']}
+		 * @type {import('focus-trap').FocusTargetValueOrFalse}
 		 */
 		setReturnFocus: {
 			required: false,
+			default: undefined,
+			type: [Object, String, Function, Boolean],
```

Comment:
```suggestion
			type: [Object, String, Function, Boolean],
			default: undefined,
```
I believe the `required` can be removed now?

Reply:
Fixed it